### PR TITLE
W-13791949 publish pdk to exchange gr

### PR DESCRIPTION
**Pull request by glenn-rodgers-sf**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: gateway/1.5/modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -80,7 +80,7 @@
 **** xref:policies-pdk-compile-policies.adoc[Compile Custom Policies]
 **** xref:policies-pdk-debug-local.adoc[Debug Custom Policies Locally]
 *** xref:policies-pdk-deploy-policies.adoc[Deploy Custom Policies]
-**** xref:policies-pdk-publish-policies.adoc[Publish Custom Policies to Exchance]
+**** xref:policies-pdk-publish-policies.adoc[Upload Custom Policies to Exchance]
```
Comment (NachoRaik) [inline gateway/1.5/modules/ROOT/nav.adoc]:
```suggestion
**** xref:policies-pdk-publish-policies.adoc[Upload Custom Policies to Exchange]
```

File: gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc [hunk 2]
```diff
@@ -1,5 +1,68 @@
...
+
+After publishing the asset, a new asset appears in Exchange. The asset's group ID, asset ID, and asset version (GAV) appears similar to the following:
+
+* *groupID*: `<organizationId>`
+* *assetID*: `my-custom-policy-dev` +
+When publishing a policy, `-dev` appears appended to the *assetID*.
+* *version*: `1.0.0-20230618115723` +
+When publishing a policy, a timestamp appears appended to the *version*.
+
+Rerunning the publish command creates a new version of the policy asset with a updated timestamp. Previously published policy versions are still avaliable.
```
Comment (NachoRaik) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc]:
```suggestion
Rerunning the publish command creates a new version of the policy asset with an updated timestamp. Previously published policy versions are still avaliable.
```

File: gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc [hunk 3]
```diff
@@ -1,5 +1,68 @@
...
+make release
+----
+
+After releasing the asset in Exchange, the asset's GAV appears similar to the following:
+
+* *groupID*: `<organizationId>`
+* *assetID*: `my-custom-policy`
+* *version*: `1.0.0`
+
+After releaseing the asset, rerunning the publish or release command results in a conflict error.
```
Comment (NachoRaik) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc]:
```suggestion
After releasing the asset, rerunning the publish or release command results in a conflict error.
```

Also, the publish won't result in conflict, at the moment.

File: gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc [hunk 4]
```diff
@@ -1,5 +1,68 @@
...
+
+To begin working on the next version of your policy, update the version number in the `Cargo.toml` file.
+
+For example, if the latests version is `1.0.0`, update the Cargo.toml` file as follows to begin working on a minor release:
```
Comment (NachoRaik) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc]:
```suggestion
For example, if the latest version is `1.0.0`, update the `Cargo.toml` file as follows to begin working on a minor release:
```

**Review by NachoRaik — CHANGES_REQUESTED** [review]

File: gateway/1.5/modules/ROOT/nav.adoc [hunk 5]
```diff
@@ -80,7 +80,7 @@
 **** xref:policies-pdk-compile-policies.adoc[Compile Custom Policies]
 **** xref:policies-pdk-debug-local.adoc[Debug Custom Policies Locally]
 *** xref:policies-pdk-deploy-policies.adoc[Deploy Custom Policies]
-**** xref:policies-pdk-publish-policies.adoc[Publish Custom Policies to Exchance]
+**** xref:policies-pdk-publish-policies.adoc[Upload Custom Policies to Exchange]
```
Comment (hannanelson) [inline gateway/1.5/modules/ROOT/nav.adoc]: If you leave the [] blank in xrefs, they get filled in automagically during the build - and you don't have to maintain them manually.

File: gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc [hunk 6]
```diff
@@ -1,5 +1,68 @@
...
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:imagesdir: ../assets/images
\ No newline at end of file
+:imagesdir: ../assets/images
+
+During the custom policy development cycle, you can upload your policy to Exchange to distribute it and manage its version. For more information about Exchange, see xref:exchange::index.adoc[].
+
+When uploading your policy to Exchange you can choose to do either of the following:
```
Comment (hannanelson) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc]:
```suggestion
When uploading your policy to Exchange, you can choose to either:
```

File: gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc [hunk 7]
```diff
@@ -1,5 +1,68 @@
-= Publishing Custom Policies to Exchange
+= Upload Custom Policies to Exchange
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:imagesdir: ../assets/images
\ No newline at end of file
+:imagesdir: ../assets/images
+
+During the custom policy development cycle, you can upload your policy to Exchange to distribute it and manage its version. For more information about Exchange, see xref:exchange::index.adoc[].
```
Comment (hannanelson) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc]: flip to put customer task (distribute/manage) first.

File: gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc [hunk 8]
```diff
@@ -1,5 +1,68 @@
...
+
+After you upload your policy to Exchange, use the following tutorials to apply your policy to a Flex Gateway instance:
+
+* xref:policies-pdk-local-deploy.adoc[]
+* xref:policies-pdk-conn-deploy.adoc[]
+
+[[publish]]
+== Publish a Custom Policy
+
+To publish a custom policy, run the publish command from the policy's root folder:
```
Comment (hannanelson) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc]: Does publish need a font change?

File: gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc [hunk 9]
```diff
@@ -1,5 +1,68 @@
...
+
+[source,ssh]
+----
+make publish
+----
+
+After publishing the asset, a new asset appears in Exchange. The asset's group ID, asset ID, and asset version (GAV) appears similar to the following:
```
Comment (hannanelson) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc]:
```suggestion
After publishing the asset, a new asset appears in Exchange. The asset's group ID, asset ID, and asset version (GAV) appear, similar to the following:
```

File: gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc [hunk 10]
```diff
@@ -1,5 +1,68 @@
...
+
+* *groupID*: `<organizationId>`
+* *assetID*: `my-custom-policy-dev` +
```
Comment (hannanelson) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc]: Global - put the + on a line by itself (it affects the line spacing). 
https://confluence.internal.salesforce.com/display/MTDT/Formatting+with+Asciidoc#FormattingwithAsciidoc-ListContinuation(akaIndentedParagraphs)

File: gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc [hunk 11]
```diff
@@ -1,5 +1,68 @@
...
+When publishing a policy, `-dev` appears appended to the *assetID*.
+* *version*: `1.0.0-20230618115723` +
+When publishing a policy, a timestamp appears appended to the *version*.
+
+Rerunning the publish command creates a new version of the policy asset with an updated timestamp. Previously published policy versions are still avaliable.
```
Comment (hannanelson) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc]:
```suggestion
Rerunning the publish command creates a new version of the policy asset with an updated timestamp. Previously published policy versions are still available.
```

Comment (hannanelson) [reply gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc]: Should publish be mono bold?
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-boldfont

Comment (hannanelson) [reply gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc]: Make sure to run vale!

File: gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc [hunk 12]
```diff
@@ -1,5 +1,68 @@
...
+
+
+[[release]]
+== Release a Custom Policy
+
+To release a custom policy, run the release command from the policy's root folder:
```
Comment (hannanelson) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc]: mono bold?

File: gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc [hunk 13]
```diff
@@ -1,5 +1,68 @@
...
+
+[source,ssh]
+----
+make release
+----
+
+After releasing the asset in Exchange, the asset's GAV appears similar to the following:
```
Comment (hannanelson) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc]:
```suggestion
After releasing the asset in Exchange, the asset's GAV appears, similar to the following:
```

File: gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc [hunk 14]
```diff
@@ -1,5 +1,68 @@
...
+
+* *groupID*: `<organizationId>`
+* *assetID*: `my-custom-policy`
+* *version*: `1.0.0`
+
+After releaseing the asset, rerunning the publish command results in a conflict error.
```
Comment (hannanelson) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc]: mono bold

**Review by hannanelson — APPROVED** [review]

File: gateway/1.5/modules/ROOT/nav.adoc (hunk 5 above)
Comment (glenn-rodgers-sf) [reply gateway/1.5/modules/ROOT/nav.adoc]: Ok, yeah, I was confused by this.

I'm under the impression that we use Gerunds in the titles but imperative mood in the sidebar. Leaving the xref blank would apply the gerunds to the sidebar. Do we have any docs on this?

Comment (hannanelson) [reply gateway/1.5/modules/ROOT/nav.adoc]: The guidance on gerunds is here: https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-gerundtitles
https://raw.githubusercontent.com/mulesoft/docs-team-wiki/master/content-templates/multi-task.adoc?token=GHSAT0AAAAAACCXRJMAYTMOINZSNZLHMREWZF2WIYQ

The guidance is use gerunds for "multi-tasks" topics - a file with multiple tasks. And imperative for each H2 in the file. 
Some people shorten their TOC titles, but the guidance has never been to make those imperative (AFAIK). Please let me know where you find this!

**Review by hannanelson — APPROVED** [review]

**Pull request merged**